lib/file_items_gateway.rb

```diff
@@ -6,13 +6,25 @@ class FileItemsGateway
   def all
     return [] unless File.exist?(FILE_PATH)
     File.open(FILE_PATH, 'r') do |file|
-      JSON.parse(file.read, symbolize_names: true)
+      item_builder = ItemBuilder::Item.new
+      item_builder.items = JSON.parse(file.read, symbolize_names: true)
+      [item_builder.build]
```

Comment:
does item_builder.build need to be put into an array here? as item_builder.build returns an array

Reply:
I think this is what was causing that issue where we had to use `items[0][0]` to actually access the hash!
If you remove the array around item_builder.build, you'll need to correct the tests so they're not using `.first` anymore

Reply:
this is because the all method will at some point return all of the orders, not just the last one placed

Reply:
Well that was the reasoning for the customer all method returning as an array. I'm assuming that this will be the same